clean.mk

```diff
@@ -14,7 +14,7 @@
 
 .DEFAULT_GOAL := clean
 
-D = /
+include common.mk
```

Comment:
Please use full path: `include $(TEST_ROOT)/TKG/common.mk` and move this inclusion after `ifndef TEST_ROOT` block. Apply this change to all places should solve the build failure.

Reply:
> Please use full path: `include $(TEST_ROOT)/TKG/common.mk` and move this inclusion after `ifndef TEST_ROOT` block. Apply this change to all places should solve the build failure.

Alright. But in `makeGen.mk` and `settings.mk`  the path `include $(TEST_ROOT)/TKG/common.mk` can be placed anywhere since it doesn't have `ifndef TEST_ROOT` isn't it @renfeiw ?

Reply:
In this case, you need to add this block. In the future, we can move this block of code to the top level makefile, but for now let's duplicate it in makeGen.mk and settings.mk